pkg/syncer/fullsync.go

```diff
@@ -533,6 +533,18 @@ func fullSyncCreateVolumes(ctx context.Context, createSpecArray []cnstypes.CnsVo
 				continue
 			}
 
+			for _, pv := range currentK8sPV {
```

Comment:
Do we have map of VolumeID to PV somewhere in the syncer?
traversing through all PVs for every successful create call in the full sync is not good.